product_pricelist_margins/models/product_pricelist_item.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_is_zero
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    cost = fields.Float(
+        related="product_tmpl_id.standard_price",
+        digits="Product Price",
+    )
+    margin = fields.Float(
+        compute="_compute_margin",
+        digits="Product Price",
+    )
+    margin_percent = fields.Float(
+        string="Margin (%)",
+        compute="_compute_margin",
+    )
+
+    @api.depends("fixed_price", "product_tmpl_id", "cost")
```

Comment:
It depends on a lot of field in fact : compute_price, applied_on, percent_price, "base", "price_discount", price_surcharge, price_round, price_min_margin.